pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,8 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		commit := fmt.Sprintf("%s", rev.Commit[:(len(rev.Commit)/3)])
```

Comment:
@mfojtik i think this is a good compromise, and also no `if` involved.
the user can send anything in the payload, if there is a case or not where the commit len is bigger than 0 but smaller than 7 i don't know(probably not), but the user has the power to decide and we shouldn't give him the opportunity to make the app panic.